go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/speakeasy-api/speakeasy
 
-go 1.24.0
+go 1.24.1
+
+toolchain go1.24.2
```

Comment:
Automatically being added by Go toolchain now, presumably because of `openapi-generation`.